stable_diffusion/requirements.txt

```diff
@@ -0,0 +1 @@
+mac_requirements.txt
```

Comment:
This is a symlink `requirements.txt` => `mac_requirements.txt` (which looks like the minimal requirement set).

Keeps things as simple as possible for now for the check.